app/controllers/catalog_controller.rb

```diff
@@ -1422,7 +1422,7 @@ def rearrange_groups_array
 
   def get_available_resources(kls)
     @edit[:new][:available_resources] = {}
-    kls.constantize.all.each do |r|
+    kls.constantize.where("type is null or type != 'ServiceTemplateAnsiblePlaybook'").each do |r|
```

Comment:
@kbrock Good point.  @h-kataria So this line becomes:
`kls.constantize.where("type is null or type != 'ServiceTemplateAnsiblePlaybook'").select(:id, :name).each do |r|`